src/main/docs/guide/httpServer/serverConfiguration/threadPools/atBlocking.adoc

```diff
@@ -0,0 +1,21 @@
+You can use the ann:core.annotation.Blocking[] annotation to mark methods as blocking.
+
+If you set `micronaut.server.thread-selection` to `AUTO`, The Micronaut Framework offloads the execution of methods annotated with `@Blocking` to the IO thread pool (See: api:io.micronaut.scheduling.TaskExecutors[]).
+
+NOTE: `@Blocking` only works if you are using `AUTO` thread selection. Micronaut Framework defaults to `MANUAL` thread selection since Micronaut 2.0. We recommend the usage of ann:scheduling.annotation.ExecuteOn[] annotation to execute the blocking operations on a different thread. `@ExecutesOn` works for both `MANUAL` or `AUTO` thread selection.
```

Comment:
```suggestion
NOTE: `@Blocking` only works if you are using `AUTO` thread selection. Micronaut defaults to `MANUAL` thread selection since Micronaut 2.0. We recommend the usage of ann:scheduling.annotation.ExecuteOn[] annotation to execute the blocking operations on a different thread pool. `@ExecutesOn` works for both `MANUAL` and `AUTO` thread selection.
```

Reply:
I have incorporated most of your suggestions here: https://github.com/micronaut-projects/micronaut-core/pull/7228/commits/e33ddf10c1321a439bae20b0d7f40ad6ebef9878

Reply:
There are gazillion places where simply "Micronaut" is used. Enforcing "Micronaut Framework" in every occurrence in the documentation is quite inappropriate, to say the least.